lib/question.js

```diff
@@ -1117,6 +1117,9 @@ module.exports = {
             urls.baseUrl = urlPrefix;
         }
 
+        const workspaceUrl = urlPrefix + '/workspace/' + variant.workspace_id;
```

Comment:
I don't believe we actually want `urlPrefix` to be used here. Depending on context, this can assume a lot of different values:
* `/pl/course/1`
* `/pl/course_instance/1`
* `/pl/course_instance/1/instructor`
I was thinking that workspaces would always be served from `/pl/workspace/[id]`, although that may be wrong. cc @mwest1066